tests/ImageSharp.Tests/Formats/Pbm/PbmMetadataTests.cs

```diff
@@ -83,12 +83,9 @@ public void Identify_DetectsCorrectComponentType(string imagePath, PbmComponentT
     }
 
     [Fact]
-    public void Identify_HandlesCraftedDenialOfServiceString()
+    public void Identify_EofInHeader_ThrowsInvalidImageContentException()
     {
         byte[] bytes = Convert.FromBase64String("UDEjWAAACQAAAAA=");
-        ImageInfo info = Image.Identify(bytes);
-        Assert.Equal(default, info.Size);
-        Configuration.Default.ImageFormatsManager.TryFindFormatByFileExtension("pbm", out IImageFormat format);
-        Assert.Equal(format!, info.Metadata.DecodedImageFormat);
+        Assert.Throws<InvalidImageContentException>(() => Image.Identify(bytes));
```

Comment:
It doesn't really matter but I don't think decoding makes much sense when the EOF is right in the header.

Reply:
Yeah, I'm happy with that.